### ISSUE TITLE
When pasting, nothing happens

### ISSUE THREAD

**Issue by hgaronfolo** [bigyuki/markdown-image-helper#2]
On Windows 7:
1. Copy image to clipboard
2. switch to Atom
3. paste image
Nothing happens.

Also tried writing ![test]( .. and then CTRL+V. 
Nothing happens.

I am not sure if I am doing something wrong? Isn't the above flow, how this is supposed to work?

Thanks for your help.

**Comment by bigyuki** [comment]
@hgaronfolo are you paste in markdown file ?

**Comment by hgaronfolo** [comment]
@bigyuki I tested a bit more.. At home I use Linux Mint, and pasting in a md file in Atom on Linux works fine (url inserted, asset folder created). However, at work on my Win7 laptop with Atom (editing an md file), the pasting doesn't work. Nothing happens when I press CTRL-V.

Can you confirm which platforms the package works on? Is it mac/linux only?

**Comment by hgaronfolo** [comment]
@bigyuki Okay, I finally figured it out :-) On my Linux machine the language in the lower right corner is set to "Github Markdown" while on Windows Atom, it is just "Markdown".

For markdown-image-helper to work, the language needs to be "Git Markdown", so on Windows Atom, I went in to settings, packages and searched for "language-gfm" (core package). This package was disabled by default in Windows Atom, but enabled by default in Linux Atom.

I enabled "language-gfm", and now I can choose "Github Markdown" in Windows Atom and markdown-image-helper now works! :-)
